src/views/HomeRedesign/HomeSection.tsx

```diff
@@ -45,7 +55,7 @@ export const HomeSection: FunctionComponent<HomeSectionProps> = ({
 
   return (
     <Fragment>
-      <Heading as="h3" color="blue.800" size="md">
+      <Heading as="h3" color="blue.800" marginY="4" size="md">
```

Comment:
Pro tip, for margin and padding you can use the following short-hands
`m` - `margin`
`mx` - `marginX`,
`my` - `marginY`,
`ml`, `mr`, `mt`, `mb` - marginLeft, right top, and bottom

Reply:
Ah cool, I knew some of those worked with tailwind css so it makes sense a lot of other frameworks also use some of the same shorthands...